src/Uno.UWP/ContextHelper.cs

```diff
@@ -26,10 +28,12 @@ public static Android.Content.Context Current
 			{
 				if (_current == null)
 				{
-					throw new InvalidOperationException(
-						"ContextHelper.Current not defined. " +
-						"For compatibility with Uno, you should ensure your `MainActivity` " +
-						"is deriving from Windows.UI.Xaml.ApplicationActivity.");
+					LogExtensionPoint.AmbientLoggerFactory
+						.CreateLogger(nameof(ContextHelper))
```

Comment:
@jeromelaban You know a better way to access the logger?

Reply:
`typeof(ContextHelper).Log().Warn();`